pkg/logs/schedulers/ad/scheduler_test.go

```diff
@@ -6,6 +6,10 @@
 package ad
 
 import (
+	"fmt"
+	pkgconfig "github.com/DataDog/datadog-agent/pkg/config"
```

Comment:
Please move the "github.com/DataDog/..." lines lower down so they are grouped with the similar includes.